src/main.cpp

```diff
@@ -25,27 +24,17 @@ int main(int argc, char* argv[]) {
         return app.exit(e);
     }
 
-    if (print_version) {
-        std::cout << version();
-        return 0;
-    }
-
     log("Iris Startup");
     rawterm::enter_alt_screen();
     rawterm::enable_raw_mode();
     rawterm::enable_signals();
 
-    try {
-        Editor e;
-        if (!file.empty()) {
-            e.init(file);
-        }
-        e.start_controller();
-        log("Iris shutdown");
-    } catch (const std::exception& e) {
```

Comment:
I was hoping to use this catch statement so I can log any exceptions in the whole app. I don’t think it works though